docs/user_guide/plugins.rst

```diff
@@ -0,0 +1,4 @@
+.. _user-plugins:
+
+User plugin guide
+=================
```

Comment:
What should go here? Even if we have nothing to say for now, a statement of intent would be nice.